compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/repositories/RemotePackageRepository.java

```diff
@@ -314,6 +314,14 @@ private PackageNameResolutionRequest toPackageNameResolutionRequest(List<ImportM
 
             if (versionCompatibilityResult.equals(SemanticVersion.VersionCompatibilityResult.GREATER_THAN)) {
```

Comment:
Do we need to have this old comparison logic too?? Can we just use the one in AbstractPackageRepository??

Reply:
+1 the idea was to use the new function for comparison